src/application/queries/milo/get-detail-session-jeune.milo.query.handler.db.ts

```diff
@@ -83,10 +82,16 @@ export class GetDetailSessionJeuneMiloQueryHandler extends QueryHandler<
       jeune.idPartenaire,
       { debut: dateSession, fin: dateSession }
     )
-    const inscription = isSuccess(resultListeSessions)
-      ? resultListeSessions.data.sessions.find(
-          session => session.session.id.toString() === query.idSession
-        )!.sessionInstance
```

Comment:
c'est ce `!` qui faisait l'erreur c'est ça ?

Reply:
c'est la regle qui foutait la merde oui , on disait que si isSuccess(resultListeSessions), 
on aura forcement une des valeur de retour qui va correspondre a query.idSession , et qui va avoir un 
. sessionInstance , alors que c'est pas le cas , d'ou le cannot read property sessionInstance of undefined